styles.css

```diff
@@ -1 +1,94 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+* {
+    background-color: #f8e3a1;
+}
+
+.container {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 50px 150px 100px auto 50px;
+        grid-template-areas:
+        "a c c"
+        "b c c"
+        "b e d"
+        "b e ."
+        "f f f";
+    gap: 16px;
+}
+
+#a {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    grid-area: a;
+    background-color: #ba3a46;
+    border-style: solid;
+    border-color: darkred;
+    color: #f8e3a1;
+}
+
+#b {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    grid-area: b;
+    background-color: #ba3a46;
+    border-style: solid;
+    border-color: darkred;
+    color: #f8e3a1;
+}
+#c {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    grid-area: c;
+    background-color: #ba3a46;
+    border-style: solid;
+    border-color: darkred;
+    color: #f8e3a1;
+}
+#d {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    grid-area: d;
+    background-color: #ba3a46;
+    border-style: solid;
+    border-color: darkred;
+    color: #f8e3a1;
+}
+#e {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    grid-area: e;
+    background-color: #ba3a46;
+    border-style: solid;
+    border-color: darkred;
+    color: #f8e3a1;
+}
+#f{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    grid-area: f;
+    background-color: #ba3a46;
+    border-style: solid;
+    border-color: darkred;
+    color: #f8e3a1;
+}
+
+@media screen and (max-width: 576px){
```

Comment:
Max breedte van 576px. Het is handiger om groter op te schalen. min-width van 576px. Dus alles groter dan dat krijgt een andere vorm.